h/views/admin_groups.py

```diff
@@ -112,6 +113,19 @@ def read(self):
         self._update_appstruct()
         return self._template_context()
 
+    @view_config(request_method='POST',
+                 route_name='admin_groups_delete')
+    def delete(self):
+        group = self.group
+        svc = self.request.find_service(name='delete_group')
+
+        svc.delete(group)
+        self.request.session.flash(
+            'Successfully deleted group %s' % (group.name), 'success')
```

Comment:
This string ought to be marked for translate-ability using `_` (see references to `TranslationString` and `_('...')`) in other calls to `session.flash`.